components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -301,6 +308,16 @@ export default class EditPostModal extends React.PureComponent {
                     />
                 </span>
             );
+
+            // Custom container for emoji picker to support fullscreen
+            // on mobile screens < 480 and positioning within the edit modal
+            // on screens > 480
+            emojiPickerContainer = (
+                <div
+                    ref='emojiPickerContainer'
+                    className='modal-emoji-container'
+                />
+            );
```

Comment:
I wonder if there's a way not put a placeholder like above. We have `import {isMobile} from 'utils/utils'` that you may use inside a component and use `isMobile` to conditionally assign the CSS class as required.

Reply:
So, I ended up adding a class (`edit-modal-body--add-reaction`) to the modal body container while the emoji window is open to temporarily update the `position: relative` to `position:static` for screens < 480px to allow the emoji window to use the parent container of the modal body for positioning and sizing.

This needed to only apply to screen sizes < 480px and it looks like the `isMobile` utility function is < 768px. Let me know if you'd like me to still consider using `isMobile`.

Dedicated `modal-emoji-container` has been removed. :)